alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ClientCertificateModel.java

```diff
@@ -0,0 +1,45 @@
+package com.synopsys.integration.alert.common.persistence.model;
+
+import java.util.UUID;
+
+import com.synopsys.integration.alert.api.common.model.AlertSerializableModel;
+
+public class ClientCertificateModel extends AlertSerializableModel {
+    private UUID id;
+
+    private String alias;
+
+    private UUID privateKeyId;
+
+    private String certificateContent;
+
+    private String lastUpdated;
+
+    public ClientCertificateModel(UUID id, String alias, UUID privateKeyId, String certificateContent, String lastUpdated) {
+        this.id = id;
+        this.alias = alias;
+        this.privateKeyId = privateKeyId;
+        this.certificateContent = certificateContent;
+        this.lastUpdated = lastUpdated;
+    }
+
+    public UUID getId() {
+        return id;
+    }
+
+    public String getAlias() {
+        return alias;
+    }
```

Comment:
After thinking about this model, I don't think exposing the alias, nor requiring it in the constructor makes sense. This field will be ignored if passed in as part of a REST request and the results if an end user GETs the model will also not matter. As all the usages of the alias are internal to Alert we should not expose this in the API. In the future, if we are required to support multiple certificates and aliases' then we can always make an additive change to include it back in without causing breaking changes.

I think that we should remove the alias from the model, but still keep it in still in the database entity class.